### PR TITLE
SOCIALPLAT-258-379: aggregating logs to Cloudwatch

### PR DESCRIPTION
**Pull request by katerinachinnappan**
## Goal
* Aggregating server logs into Cloudwatch
* Remove `AWS_XRAY` container

## References

JIRA ticket:
* [https://mozilla-hub.atlassian.net/browse/SOCIALPLAT-379](https://mozilla-hub.atlassian.net/browse/SOCIALPLAT-379)

### CODE REVIEW

File: .aws/src/main.ts [hunk 1]
```diff
@@ -320,18 +329,6 @@ class AnnotationsAPI extends TerraformStack {
             },
           ],
         },
-        {
```
Comment (bassrock) [inline .aws/src/main.ts:323]: Is Xray removed from the code base?

**Review by bassrock — APPROVED** [review]
LGTM assuming my comment checks out

File: .aws/src/main.ts (hunk 1 above)
Comment (katerinachinnappan) [reply .aws/src/main.ts:323]: @bassrock yep! removed it in the previous PR

**Review by jpetto — APPROVED** [review]
nice!

**Pull request merged**